app/assets/templates/pages/home.html

```diff
@@ -36,81 +36,97 @@
-        </li>
-        <li class="list-group-item">
-          <div>Duration: {{duration}} mins</div>
-          <div slider config="sliderDurationConfig" duration="duration" id="duration-slider" data-name="duration"></div>
-        </li>
-      </ul>
+      </li>
+      <li class="list-group-item" ng-show="outBox">
+        <div>Difficulty Level: {{difficulty}}</div>
+        <div difficulty-Slider config="sliderDifficultConfig" model="difficulty" id="difficulty-slider" data-name="difficulty"></div>
+      </li>
+      <li class="list-group-item" ng-show="outBox">
+        <div>Scenery Level: {{scenery}}</div>
+        <div scenery-Slider config="sliderSceneryConfig" model="scenery" id="scenery-slider" data-name="scenery"></div>
+      </li>
+      <li class="list-group-item" ng-show="outBox">
+        <div>Distance: {{distance}} km</div>
+        <div distance-Slider config="sliderDistanceConfig" model="distance" id="distance-slider" data-name="distance"></div>
+      </li>
+      <li class="list-group-item" ng-show="outBox">
+        <div>Duration: {{duration}} mins</div>
+        <div slider config="sliderDurationConfig" duration="duration" id="duration-slider" data-name="duration"></div>
+      </li>
+    </ul>
   </div>
-
-<!--   <button class="btn btn-sm btn-default close-button" ng-click="showPanel()">Close</button> -->
-
   <div id="info-panel" class="container-fluid" ng-show="true">
     <div class="search-results" ng-show="search">
-      <h1 ng-click="eee()">Search Results</h1>
-
-      <div ng-repeat="trail in trails2.trails" class="list-group-item" ng-click="switchPanel()">
-        <img src="http://www.oasistrek.com/{{trail.photo_urls[0]}}" class="search-result-pix col-md-3"></img>
-        <div class="search-results-item col-md-7"><h2>{{trail.name}}</h2>{{info.climate}}</div>
-        <div class="btn glyphicon glyphicon-chevron-right  search-results-button col-md-2"></div>
+      <h1>Search Results...</h1>
+      <div ng-repeat="trail in trails" class="list-group-item trail-results" ng-click="switchPanel()">
+        <img src="http://www.oasistrek.com/{{trail.photo_urls[0]}}" class="search-result-pix col-md-3">
+        <div class="search-results-item col-md-7">
+          <h2>{{trail.name}}</h2>
+          <ul ng-bind-html="makeStars(trail.difficulty)"></ul>
+          <!-- {{info.climate}} -->
+        </div>
+        <button class="btn glyphicon glyphicon-chevron-right search-results-button col-md-2"></button>
       </div>
-
-    </div>
-    
-    <div class="one-trail" ng-show="trail">
-      <div>
-        <button ng-click="switchPanel()" id="back-button" class="btn glyphicon glyphicon-chevron-left"></button>
-        <h1 class="title-name">Trail Name</h1>
+    </div>    
+    <div id="one-trail" ng-show="trailPanel">
+      <div class="list-group-item" ng-click="switchPanel()">
+        <button id="back-button" class="btn glyphicon glyphicon-chevron-left"></button>
+        <h2 class="title-name">{{trail.name}}</h2>
```

Comment:
ng-show=trail is deleted here
and changed to ng-show=trailPanel

basically it is a switch